Scripts/run

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
```

Comment:
This file is pretty strange. Is it necessary? Same for `upload-symbols`

Reply:
No, but it simplifies the usage. Firebase Crashlytics framework builded with carthage should have this files in the package, but it don't have.